govgen-1/README.md

```diff
@@ -0,0 +1,35 @@
+# 🔗 `govgen-1`
+
+![chain-id](https://img.shields.io/badge/chain%20id-govgen--1-blue?style=for-the-badge)
+
+## Register in the Genesis
+
+To register your validator node in the `genesis.json` you just need to provide a signed `gentx` with an initial delegation of `10000000000ugovgen`.
+
+```sh
+# Init node
+govgend init your-moniker --chain-id govgen-1
+
+# Create keys, be careful with the mnemonic 👀
+govgend keys add your-key-name
+
+# Set account necessary balance
+govgend add-genesis-account your-key-name 10000200000ugovgen
```

Comment:
```suggestion
govgend add-genesis-account your-key-name 10000000ugovgen
```
we recommend using this number because it will reflect the actual balance you will have on GovGen. However, keep in mind you can change this number however you won't and it won't affect your actual balance on GovGen, this is only needed for you in order to be able to run the `gentx` command locally. Just make sure you are using a number that is bigger than 1000000ugovgen (1 $GOVGEN which is the minimum possible self delegation)